database/pdb.ksy

```diff
@@ -0,0 +1,964 @@
+meta:
+  id: rekordbox_pdb
```

Comment:
file name should match its `id`

Reply:
If this is true, it should be stated in the style guide.

Reply:
It is, as far as I remember.

Reply:
I searched the whole style guide before posting my comment, and could not find anything along those lines. I will rename the files when I have a few minutes though.

Reply:
All right, I have renamed the files and submitted a PR to the style guide to help future me avoid making the same mistake. 😄 Thanks. I still need to fix something in the `rekordbox_anlz.ksy` file to avoid Java exceptions when the file contains a `fourcc` that we have not previously encountered, as I am currently discussing in the gitter channel.

Reply:
Ok, I have revised the mapping to stop trying to use enums until the underlying compiler issue is resolved.

Reply:
Any chance we can do `devicesql_pdb` here, as we know that it is not exactly specific to rekordbox?

Reply:
There was another thread on this topic, @GreyCat. The filename started out not being specific to rekordbox, band @KOLANICH suggested the current rename. I was hesitant at first but ended up agreeing because, while there are certainly aspects of the format which are generally applicable to other DeviceSQL implementations, the specific column types which are used by this mapping (e.g. tracks, artists, playlists, etc.) are clearly used only by rekordbox.

Reply:
Ok, I agree with `rekordbox_pdb`, but please rename the file as `database/rekordbox_pdb.ksy` then?

Reply:
>while there are certainly aspects of the format which are generally applicable to other DeviceSQL implementations

Can they be moved into a separate ksy and used via import mechanisms?